main/main.js

```diff
@@ -116,23 +116,38 @@ function openTabInWindow (url) {
 
 function handleCommandLineArguments (argv) {
   // the "ready" event must occur before this function can be used
+  var initTaskQuery = undefined
   if (argv) {
+
+    // check for -t task query
+    if (argv.includes('-t') && argv.indexOf('-t') > 0){
+      // query for specific task to add search to
+      initTaskQuery = argv[argv.indexOf('-t') + 1]
```

Comment:
I think we can assume that IPC messages will be received in the order they're sent, in which case we can just send a `switchToTask` IPC here, and then leave the code below as `switchToTab` without any task information. That should simplify this a fair amount.